easybuild/easyconfigs/g/GLib/GLib-2.60.0-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MesonNinja'
+
+name = 'GLib'
+version = '2.60.0'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['20865d8b96840d89d9340fc485b4b1131c1bb24d16a258a22d642c3bb1b44353']
+
+builddependencies = [
+    ('Meson', '0.50.0', '-Python-3.7.2'),
+    ('Ninja', '1.9.0'),
+    ('binutils', '2.31.1'),
+    ('Python', '3.7.2'),
+    ('pkg-config', '0.29.2'),
+]
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.8.1'),
+    ('libxml2', '2.9.8'),
+    ('PCRE', '8.43'),
+    ('util-linux', '2.33'),
+]
+
+# avoid using hardcoded path to Python binary in build step
+preconfigopts = "export PYTHON=python && "
+
+configopts = "--buildtype=release --default-library=both "
+configopts += "-Dsystemtap=false"
```

Comment:
Need -Dlibdir=lib or the libs will end up in lib/x86_64-linux-gnu/ on debian/ubuntu multiarch systems.

Reply:
We're probably going to fix this in the measonninja.py instead.

Reply:
See https://github.com/easybuilders/easybuild-easyblocks/pull/1673